portfolio/pom.xml

```diff
@@ -23,6 +23,18 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+      <version>2.8.6</version>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.appengine</groupId>
+  		<artifactId>appengine-api-1.0-sdk</artifactId>
```

Comment:
I wonder if there is a better text editor you could use, or settings to handle this better, but I wanted to point out the mixing of spaces/tabs here leading to inconsistent indentation again. I won't block approval on this but it'd be nice to clean up